tests/bearlib/aspects/DecoratorsTest.py

```diff
@@ -35,33 +60,73 @@ def test_mapping(self):
             get_aspect('Clone')('py', min_clone_tokens=30),
         ])
         result = self.bear.execute()
-        self.assertEqual([True, 30], result)
+        expected = self.EXPECTED.copy()
+        expected['minimum_clone_tokens'] = 30
+        expected['remove_unreachable_code'] = True
+        self.assertResult(expected, result)
 
     def test_setting_priority(self):
         self.section.aspects = AspectList([
             get_aspect('UnreachableCode')('py'),
             get_aspect('Clone')('py', min_clone_tokens=30),
         ])
-        self.section.append(
-            Setting('remove_unreachable_code', 'False'))
-        self.section.append(
-            Setting('minimum_clone_tokens', 40))
+        self.section.append(Setting('remove_unreachable_code', 'False'))
+        self.section.append(Setting('minimum_clone_tokens', 40))
```

Comment:
These lines are not part of your change. Do not modify their style unless you need to improve their functionality as well.
It just makes your patch large and 'steals' `git blame`

Reply:
Yeah, I guess this will make a good newcomer issue :)